fossid-webapp-client/src/main/kotlin/api/summary/Summarizable.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid.api.summary
+
+/**
+ * This interface marks FossId file results that can be included in a scan summary
+ */
+interface Summarizable {
+    /**
+     * Convert the current FossId file to an Object suitable for a scan summary
+     */
+    fun toSummary(): SummaryIdentifiedFile
+
+    fun getFileName(): String
+
+    fun getCopyright(): String?
```

Comment:
If you add docs, please add docs for all of the functions.

Reply:
done